addons/account/models/chart_template.py

```diff
@@ -141,7 +141,7 @@ def try_loading(self, template_code, company, install_demo=True):
         :type install_demo: bool
         """
         if not company:
-            company = self.env.company
+            return
```

Comment:
Just to understand, what is the use-case and why change?

Reply:
I didn't find any valid use case
The change is because of the search done in `account_demo.xml`, when the domain `[('partner_id.country_id.code', 'in', ['US', False])]` doesn't return anything, we don't want to fallback on the main company

Reply:
Ah, I see, thank you!
It could use a small comment, so people who read this function also get to know about it. :)